stratum/hal/lib/p4/p4_info_manager.h

```diff
@@ -280,13 +281,16 @@ class P4InfoManager {
   P4ResourceMap<::p4::config::v1::Meter> meter_map_;
   P4ResourceMap<::p4::config::v1::ValueSet> value_set_map_;
   P4ResourceMap<::p4::config::v1::Register> register_map_;
+  P4ResourceMap<::p4::config::v1::Extern> extern_map_;
 
   // These containers verify that all P4 names and IDs are unique across all
   // types of resources that have an embedded Preamble.
   absl::flat_hash_set<uint32> all_resource_ids_;
   absl::flat_hash_map<std::string, const ::p4::config::v1::Preamble*>
       all_resource_names_;
   absl::flat_hash_map<uint32, std::string> id_to_resource_type_map_;
+  google::protobuf::RepeatedPtrField<p4::config::v1::Meter> meter_objects;
```

Comment:
I think this can be removed and replaced in the function where it is used, by a local variable to contain this. This over here is only serving use for temporary purpose. Doesn't make sense adding it here in the object